### PR TITLE
update to eh1

### PR DESCRIPTION
**Pull request by romancardenas**
Tested with a SiFive E310x chip. Works as expected

### CODE REVIEW

**Comment by romancardenas** [comment]
Hmm, that's weird, I haven't added any lines, coverage should have remained the same...

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [1.52.1]
+        rust: [stable]
```
Comment (eldruin) [inline .github/workflows/build.yml]:
```suggestion
        rust: [1.60.0]
```
I prefer to keep clippy pinned because the hints change frequently.

**Review by eldruin — COMMENTED** [review]
Thank you so much!
This looks good to me. Don't worry about the coverage change.
Could you add an entry to the changelog about the change and also about the change to the MSRV?

**Review by eldruin — APPROVED** [review]
Thank you!

**Pull request merged**